File: works/tmp/10869.js
let fs = require('fs');
let input = fs.readFileSync('/dev/stdin').toString().split(' ');

let a = Number(input[0]);
let b = Number(input[1]);

console.log("%d", a+b);
console.log("%d", a-b);
console.log("%d", a*b);
console.log("%d", Math.floor(a/b));
console.log("%d", a%b);

File: works/tmp/10869.dart
import 'dart:io';

void main()
{
    var input = stdin.readLineSync()!.split(' ');
    int A = int.parse(input[0]);
    int B = int.parse(input[1]);
    print(A+B);
    print(A-B);
    print(A*B);
    print((A/B).floor());
    print(A%B);
}